grammarinator/process.py

```diff
@@ -621,6 +632,7 @@ def __init__(self, lang, work_dir=None):
         env = Environment(trim_blocks=True,
                           lstrip_blocks=True,
                           keep_trailing_newline=False)
+        env.filters['repr'] = lambda s: repr(s)
```

Comment:
Well, it was me suggesting using the `repr` filter, which turned out not to exists as a built-in filter in Jinja. But there is, according to documentation, an alternative that should work without externally defining `repr`. As operator `%` is available in template expressions `"%r" % foo` shoud work out of the box.

But, here comes the but. The current way the template handles literals looks like as follows:

```py
{% macro processLiteralNode(node) %}
UnlexerRule(src='{{ node.src }}', parent=current)
{% endmacro %}
```

I.e., no "repr" or "%r" or anything like that is used, but `node.src` is simply surrounded by single quotes. I'm not completely sure why that works, why there is no need for any escaping, e.g., for a single quote literal. But if we want to maintain compatibility with the existing approach, then whenever we output a literal, we should use the same approach. (And if we find a better approach, it has to be applied at all places where literals are output.)